import-mailbox-to-gmail.py

```diff
@@ -247,7 +310,8 @@ def process_mbox_files(username, service, labels):
             message.replace_header('Message-ID', msgid)
         except Exception:
           logging.exception('Failed to fix brackets in Message-ID header')
-        metadata_object = {'labelIds': [label_id]}
+        metadata_object = get_metadata(service, username, labels, message, label_id, labelname)
+        if not metadata_object: continue
```

Comment:
1. Please put `continue` in a new indented line.
2. When does this happen? Only if this is a spam/trash label we're skipping?